content/blog/2023-11-06-one-click-local-development-announcing-leapps-localstack-integration/index.md

```diff
@@ -0,0 +1,126 @@
+---
+title: "One Click Local Development: Announcing Leapp's LocalStack Integration"
+description: "We're excited to announce our partnership with Leapp by Noovolari to integrate LocalStack. Now you can easily create LocalStack sessions in Leapp to connect to the cloud emulator without hassles."
```

Comment:
nit: I'd rather highlight the positive aspects ("seamless switch"), rather than emphasizing a negative one ("hassle") here.

Perhaps something like:
```suggestion
description: "We're excited to announce our partnership with Leapp by Noovolari to integrate LocalStack. Now you can easily create LocalStack sessions in Leapp and seamlessly switch between local dev and remote cloud environments."
```